### ISSUE TITLE
Nothing happens when I select `Account Preferences` on certain extensions

### ISSUE THREAD

**Issue by bamurtaugh** [microsoft/vscode#231966]
Testing #229420, https://github.com/microsoft/vscode/issues/231880

```
Version: 1.95.0-insider (user setup)
Commit: 804f450ca900d24db25e7174e8b6dfb3fb2a318c
Date: 2024-10-22T13:30:10.100Z
Electron: 32.2.1
ElectronBuildId: 10427718
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.26120
```

GH Copilot: `v1.241.1169 (pre-release)`
GH Copilot Chat: `v0.22.2024102202 (pre-release)`
OS: Windows 11 Insiders

![Image](https://github.com/user-attachments/assets/c7e5d838-88c4-4aaa-8629-7bf18e54c152)

**Comment by bamurtaugh** [comment]
I experience the same issue with GitHub Repositories.

**Comment by bamurtaugh** [comment]
I can get into the account preference menu if I go via: `Account Menu -> An Account -> Manage Trusted Extensions -> Gear icon on one of the Extensions`.

Example for Live Share through this path:

![Image](https://github.com/user-attachments/assets/b08ab1b3-7128-4e8f-b02c-f555bb32b66d)
![Image](https://github.com/user-attachments/assets/d2084570-3324-438b-bc44-12516999d110)

But if I go through this path, nothing happens when I click `Account Preferences`:

![Image](https://github.com/user-attachments/assets/8a1fe169-2adc-4056-900a-9a8f6d58bd89)

**Comment by TylerLeonhardt** [comment]
The root cause is that Remote Tunnels is "trusted for BOTH GitHub & Microsoft" (for reasons) this means that:

* It's an extension that we know can have Account Preferences so we show the item in the Context Menu
* We don't know which provider to update the preference for because it has access to both GitHub & Microsoft

We could show a quick pick to ask the user which provider (GH/MS) to configure... but at this point in time, it hasn't used any account. And for the Remote Tunnels case, it _can_ ask for both of those types of auth but it really only uses one.

Because this is a bit of a corner case, I will make a more pragmatic change here.

Basically showing a dialog that says "This extension has not used any accounts yet." when you pick the Context Menu Item.

I _won't_ change the experience in the Account Menu... because there we know which auth provider we are changing.

This is all very niche so I think this difference isn't so bad... and once the extension actually uses auth this alllll doesn't matter anymore.
